helper-script.py

```diff
@@ -0,0 +1,247 @@
+import argparse
+import logging
+import os
+import re
+import shutil
+import subprocess
+import typing
+
+import requests
+from github import Auth, Github
+
+logger = logging.getLogger(__name__)
+logging.basicConfig(level=logging.INFO)
+
+WORKING_DIR = "./tmp"
+DIFF_FILE = "./diff.patch"
+
+auth = Auth.Token(os.getenv("GITHUB_TOKEN"))
+g = Github(auth=auth)
```

Comment:
Let's call this at least `gh`, or maybe `gh_client` or `github_client`. Single-letter variable names are fine for one-liners or small code blocks, but seems like a bad idea for a global.

Reply:
Initially, I did have it as "gh_client". Later I changed it to `g` because it's what the PyGithub doc uses, lol. I'll change it to either `gh_client` or `github_client` then.